### PR TITLE
update fastlane descriptions

### PR DESCRIPTION
**Pull request by IzzySoft**
As discussed in #1:

* `de/short_description.txt` added
* `en-US/full_description.txt` rephrased

The latter I also reformatted in a way that should fit all. Background:

* F-Droid applies `nl2br` to all `full_description.txt`, regardless what. In your case that would mean your bullet-point lists would have additional line-breaks included as well, which would look a bit messy
* In my repo, I control that per-app, with choices of taking it as-is (becomes a single wall-of-text in your case), applying `nl2br` as F-Droid does (same issue as there then), or handling it as Markdown (doesn't match with the old formatting)
* No idea how Play handles it.

So I learned to go with a compromise I call "Markdown lite" – which works OK with F-Droid, and fine with my repo:

* Keeping each paragraph on a single line, separated from the next paragraph by an empty line
* separating bullet-point lists by empty-lines from surrounding paragraphs
* applying "permitted inline HTML"

While F-Droid does not interprete "full Markdown", it seems to handle this OK. How Play handles it, remains to be seen (we can adjust again then, just let me know).

Please check if this is acceptable to you. Let me know if you want something changed before merging. Of course you can change everything aftger merging :laughing:

### CODE REVIEW

**Comment by mirfatif** [comment]
@IzzySoft thanks for your contribution.

**Comment by IzzySoft** [comment]
Gladly! If you need further help (in this direction; I'm no Android dev, so with that I cannot help), just let me know.

**Comment by mirfatif** [comment]
@IzzySoft 
> In your case that would mean your bullet-point lists would have additional line-breaks included as well, which would look a bit messy

That is the case, as it looks on F-Droid Android client:

 

List items have unnecessary line breaks in between. Inverted commas around "permissions" are weird. Also screenshots won't show (not sure if there is some problem at my end).

Can we fix these issues?

**Comment by IzzySoft** [comment]
> List items have unnecessary line breaks in between.

Yes, because nl2br is applied in general (no distiction at F-Droid.org as it is with my repo – but that would mean a lot of hand-work (or a good detector) to distinguish proper HTML, Markdown and "funny txt". My "Markdown lite" is a compromise here: looks better as long as the bullet points only have short text – but as soon as the line needs to be wrapped, looks not that nice anymore. But also not as ugly as `<li>` with added `<br>` :laughing:

> Also screenshots won't show

No. A long-standing issue with one of the libraries used in fdroidclient. You'll probably see the same for icons on the other tabs (which I sometimes mockingly call a "feature": makes it easy to spot the apps from my repo, as they'll have icons and screenshots). Answering your not-yet-asked question: server overload seems to cause the image fetcher to enter doze mode, per server. So if there were X timeouts from f-droid.org, it stops loading images from there – but still loads them e.g. from my server. No idea about the size of X.

TL;DR: My PR solves the former issue, but cannot do anything about the latter unfortunately.
